stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -578,7 +578,8 @@ extension ProcessTerminationStatus {
       return CInt(signal) == SIGILL || CInt(signal) == SIGTRAP
     default:
       // This default case is needed for standard library builds where
-      // resilience is enabled
+      // resilience is enabled.
+      // FIXME: Need a way to suppress when not.
       return false
```

Comment:
The warning should already be suppressed if the enum is in the same module. If it isn't there's existing logic that's broken...